Slider/Slider.jsx

```diff
@@ -27,41 +27,39 @@ export default class Slider extends MaterialComponent {
     this.MDComponent = new MDCSlider(this.base);
     this.MDComponent.listen("MDCSlider:change", this._onChange);
     this.MDComponent.listen("MDCSlider:input", this._onInput);
+    this.setValue(); // set initial value programatically because of error if min is greater than initial max
   }
   componentWillUnmount() {
     this.MDComponent.unlisten("MDCSlider:change", this._onChange);
     this.MDComponent.unlisten("MDCSlider:input", this._onInput);
     this.MDComponent.destroy && this.MDComponent.destroy();
   }
-  materialDom(allprops) {
-    const {
-      disabled = false,
-      min = 0,
-      max = 100,
-      value,
-      step,
-      tabindex = 0,
-      ...props
-    } = allprops;
+  setValue(props = this.props) {
+    const { disabled = false, min = 0, max = 100, value, step } = props;
+    if (this.MDComponent) {
+      if (min > this.MDComponent.max) {
+        this.MDComponent.max = max;
```

Comment:
max before min or min before max. it's because MDC throws an error if you set min to be greater than max.